daemon/src/model/cfd.rs

```diff
@@ -655,9 +660,10 @@ impl Cfd {
     }
 
     pub fn setup_contract(self, completed: SetupCompleted) -> Result<Event> {
-        if self.version > 0 {
+        // Version 1 is acceptable, as it means that we started contract setup
+        if self.version > 1 {
```

Comment:
We could check for the `during_contract_setup` boolean here instead? I think that would be more accurate.

Reply:
We can't really, until we have the `Accepted` event too :( 
In order to do that, we need to move the `Started` event into started() method, and that it turn makes everything jump into the contract setup state (even before the order is accepted).
Added a TODO to address in follow-up.